tests/api/images/upload.test.js

```diff
@@ -0,0 +1,47 @@
+require('should')
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const host = CONFIG.fullPublicHost()
+const someJpegPath = __.path('client', 'public/images/small/brittanystevens.jpg')
+const { authReq, getUser } = require('../utils/utils')
+const { shouldNotBeCalled } = require('root/tests/unit/utils')
+const endpoint = '/api/images?action=upload'
+const { createReadStream } = require('fs')
+const fetch = require('node-fetch')
+const FormData = require('form-data')
+const { isImageHash } = require('lib/boolean_validations')
+
+describe('images:upload', () => {
+  it('reject uploads on assets containers', async () => {
+    await authReq('post', `${endpoint}&container=assets`)
+    .then(shouldNotBeCalled)
+    .catch(err => {
+      err.body.status_verbose.should.startWith('invalid container')
+    })
+  })
+
+  it('reject uploads on remote containers', async () => {
```

Comment:
i dont understand the purpose of those 2 `reject` tests (assets & remote) if non of those parameters are in a `should` test later. Does that mean its possible to post a `container=remote` ? If yes would be nice to document it with a test, no?

Reply:
It's not possible to post a `container=remote`, no, only on `entities` and `users` containers, but `assets` is a container (where uploads are done directly by an instance admin, without passing by this endpoint) and `remote` is a pseudo-container, known by the `/img` endpoint, but that doesn't have a container in the object storage (source image files aren't fetched from the object storage but from the remote URL). Hopefully that's clearer, do you still see a missing test?

Reply:
it is clearer, but i still dont get why we test 2 rejections without any acceptation tests later. if i push the logic why not testing that `container=foo` only and not twice like now ?
In any case could we paste your comment inside the test file as a documentation, to not lose it in the limbs of gh comments?

Reply:
acceptance tests added in a7d6c3c20 and documentation added in 7ec56c963